src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -170,11 +170,10 @@ trait BackendCall {
    */
   def poll(previous: ExecutionHandle)(implicit ec: ExecutionContext): Future[ExecutionHandle]
 
-  def workflowLoggerWithCall(akkaLogger: Option[LoggingAdapter] = None) = WorkflowLogger(
-    backend.backendClassString,
+  def workflowLoggerWithCall(source: String, akkaLogger: Option[LoggingAdapter] = None) = WorkflowLogger(
+    source,
```

Comment:
I'm confused... are all these other necessary for changing the datatype to TINYINT or just stowaway changes of general use for Call Caching?